CapstoneBackend/Models/Analysis.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace CapstoneBackend.Models
+{
+    public class Analysis : Diagnostics
```

Comment:
See if there are any other data-points/ attributes that could be added to analysis.

Things that might be cool to take a look-at from the physicians perspective.

Estimated Heart-Rate? Estimated Motion (on a scale of 1-10) based on position changes (10 = bad. 1 = no motion which could be good or bad based on research?)

Any areas that are concerning (sudden spike in HR or breathign rate).